app/views/about/index.html.slim

```diff
@@ -2,54 +2,46 @@ section#header-banner.cover-about
     .banner-content.single-page.text-center
       .banner-border
         .banner-info
-          h1 About
-          p Aliquam pellentesque nunc enim
+          h1 About Us
 #page-content.container
   .section-w-image
     .row
       .col-sm-7.about-img
-        p
+        .team.block.text-center
+            = image_tag 'http://res.cloudinary.com/hollyandpine/image/upload/c_fill,h_544,w_1200/v1511282492/Product%20Images/Group%20Photos/UNADJUSTEDNONRAW_thumb_260d.jpg', alt: 'team'
       .col-sm-5.col-sm-offset-7.about-description
         h4 ABOUT THE SHOP
-        p Quisque nec mi urna. Nam nec enim id erat bibendum tincidunt. Morbi maximus neque sed ex aliquet, in lobortis mi condimentum. Vestibulum laoreet placerat ex id viverra. Ut auctor odio magna, in vulputate lorem tempus sed. Quisque bibendum molestie lacus. Phasellus quis commodo ex, id volutpat purus. Quisque ac ante a tellus auctor molestie in ac sem. Mauris non ultricies ligula. Aliquam malesuada finibus magna a egestas.
-        p Vivamus eu lacus eros. Integer ornare scelerisque sapien ac blandit. Sed blandit diam vel tincidunt placerat. Sed a eleifend dui. Cras gravida dapibus sapien et rhoncus. Nulla facilisi. Donec vitae pharetra sapien.
-        p Nullam auctor ac erat in semper. Aliquam erat volutpat. Nullam malesuada nunc vel lectus ultrices.
+        p Holly & Pine is a lifestyle company providing luxury gift styling and wrapping during the holiday season! 
+        p Nothing is more important than time spent with family, friends, and loved ones during this time of year. We know your time spent wrapping gifts could be better spent elsewhere. Let us help you during this time of giving.
+        p During this season of giving, we will donate 50¢ from every wrapped package to Toys For Tots. 
   section#team
     .container
       h2.text-center.space-top-2x THE TEAM
       .row
         .col-sm-4
           .team.block.text-center
-            = image_tag 'team.jpg', alt: 'team'
+            = image_tag 'http://res.cloudinary.com/hollyandpine/image/upload/v1511297538/Product%20Images/Headshots/daIW7ovPR1WzU_Fft_uxKw_thumb_2670.jpg', alt: 'team'
             .team-info-box
-              h6 Katharine Grace McGuillicutty Sellers
-              p.team-role CEO, Rump Shaker
+              h6 Katharine Sellers
+              p.team-role CEO, Gift Stylist & Wrapper
               p.social.text-center
-                = link_to '#'
-                  i.fa.fa-facebook
-                = link_to '#'
-                  i.fa.fa-twitter
-                = link_to '#'
-                  i.fa.fa-google-plus
+                = link_to 'https://www.instagram.com/sellerskg/'
```

Comment:
Remember to add `target: '_blank'` to any links you want to open in a new tab (like, leaving the site...)

```
 = link_to 'https://www.instagram.com/sellerskg/', target: '_blank'
```